Lib/unittest/runner.py

```diff
@@ -141,12 +155,40 @@ def __init__(self, stream=None, descriptions=True, verbosity=1,
         self.failfast = failfast
         self.buffer = buffer
         self.tb_locals = tb_locals
+        self.durations = durations
         self.warnings = warnings
         if resultclass is not None:
             self.resultclass = resultclass
 
     def _makeResult(self):
-        return self.resultclass(self.stream, self.descriptions, self.verbosity)
+        try:
+            return self.resultclass(self.stream, self.descriptions,
+                                    self.verbosity, durations=self.durations)
+        except TypeError:
+            # didn't accept the durations argument
+            return self.resultclass(self.stream, self.descriptions,
+                                    self.verbosity)
+
+    def _printDurations(self, result):
+        ls = sorted(result.collectedDurations, key=lambda x: x[1])
```

Comment:
pytest and libregrtest list the slowest tests first (descending order). I think it would be good to have slowest test at the top.

libregrtest :

https://github.com/python/cpython/blob/0ef8c157e9195df0115c54ba875a5efb92ac22fb/Lib/test/libregrtest/main.py#L348

Sample CI output :

https://travis-ci.org/python/cpython/jobs/524250451#L2322

Current PR : 

```
./python.exe -m unittest test.test_re --durations 5
.........................................................................ss..................................................
Slowest test durations
----------------------------------------------------------------------
0.166s     test_long_pattern (test.test_re.PatternReprTests)
0.188s     test_re_escape (test.test_re.ReTests)
0.190s     test_re_escape_bytes (test.test_re.ReTests)
1.004s     test_big_codesize (test.test_re.ReTests)
1.085s     test_re_tests (test.test_re.ExternalTests)

----------------------------------------------------------------------
Ran 125 tests in 3.464s

OK (skipped=2)
```

Showing the slowest tests at top like pytest and libregrtest

```
./python.exe -m unittest test.test_re --durations 5
.........................................................................ss..................................................
Slowest test durations
----------------------------------------------------------------------
1.063s     test_big_codesize (test.test_re.ReTests)
1.005s     test_re_tests (test.test_re.ExternalTests)
0.213s     test_re_escape (test.test_re.ReTests)
0.172s     test_re_escape_bytes (test.test_re.ReTests)
0.140s     test_long_pattern (test.test_re.PatternReprTests)

----------------------------------------------------------------------
Ran 125 tests in 3.426s

OK (skipped=2)
```

Reply:
Agreed. Done.